searx/exceptions.py

```diff
@@ -61,7 +61,7 @@ class SearxEngineAccessDeniedException(SearxEngineResponseException):
     """This settings contains the default suspended time (default 86400 sec / 1
     day)."""
 
-    def __init__(self, suspended_time: int = None, message: str = 'Access denied'):
+    def __init__(self, suspended_time: int = 0, message: str = 'Access denied'):
```

Comment:
Changing the default frome None to 0 would print `suspended_time=0`. Is this change going to result in a confusing message? If so, consider using `typing.Optional[int]`.